recipes/django-bakery/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "django-bakery" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "2a37e1a9d56812a2609020d3e53b91089b5833a1cebdc38b590eb2dfb12f8d2f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Please add `noarch: python`

Reply:
Thanks. I will make it noarch and re-render.